main.tex

```diff
@@ -843,41 +843,41 @@ \subsubsection{ASCII}
 % \subsubsection{\texttt{.ascii}: fast readers and writers}
 
 The \package{astropy.io.ascii} subpackage now includes a significantly faster
-Cython/C engine for reading and writing ASCII files. This is available for the
-most common formats.  On average the new engine is about 4 to 5 times faster
-than the corresponding pure-\python implementation, and is often comparable to
-the speed of the \package{Pandas} \citep{pandas} ASCII file interface.  The
+Cython/C engine for reading and writing ASCII files. This is available for most
+of the common formats.  On average, the new engine is about 4 to 5 times faster
```

Comment:
Note: I hope I didn't change the intended meaning here. Also `pandas` is all lowercase.